lib/builtIn/ForLoop.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const DefaultConfig = require('../DefaultConfig');
+const DEFAULT_CONFIG = {
```

Comment:
Could you please define this as a type called `ForLoopConfiguration` and use it in the documentation of this class?

Reply:
fixed, i hope, can you check again please?